provider/image.go

```diff
@@ -756,88 +797,79 @@ func configureDockerClient(configs map[string]string) (*client.Client, error) {
 	if val, ok := configs["certPath"]; ok {
 		certPath = val
 	}
-	if val, ok := configs["host"]; ok {
-		host = val
-	}
+
+	var err error
```

Comment:
(nit) do we need this explicit declaration? It looks like all instances in the context are assigned via `:=`, unless I missed something.